mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -109,11 +124,17 @@ CLASS net/minecraft/unmapped/C_smpzexmx net/minecraft/client/gui/DrawableHelper
 		ARG 4 color
 	METHOD m_kpibnkqc changeScissorState (Lnet/minecraft/unmapped/C_erwldarl;)V
 		ARG 0 area
-	METHOD m_ksjujvmg (Lnet/minecraft/unmapped/C_cnszsxvd;IIIIIIIIII)V
+	METHOD m_ksjujvmg drawNineSlicedTexture (Lnet/minecraft/unmapped/C_cnszsxvd;IIIIIIIIII)V
```

Comment:
hm, i wonder if nine-slice or nine-patch's the name for these thingies

Reply:
yes, nine-slice's the moj name, but i saw nine-patch being used before

Reply:
It appears that nine-slicing is the more common term for this; Will retract my question